test/metrics/prom-metrics-linter/README.md

```diff
@@ -0,0 +1,22 @@
+# CI Metrics Tests
+## prom-metrics-linter test
+### Test
+prom-metrics-linter is a common image for testing metrics follow the naming conventions in Kubevirt sub-operators projects.
```

Comment:
prom-metrics-linter is a common image for testing metrics follow the naming conventions in Kubevirt sub-operators projects. ->
prom-metrics-linter is a common image for testing that the metric names follow the naming conventions in all the Kubevirt sub-operators projects.